api/v1beta1/azuremanagedcontrolplane_webhook.go

```diff
@@ -260,6 +261,10 @@ func (mw *azureManagedControlPlaneWebhook) ValidateUpdate(ctx context.Context, o
 		allErrs = append(allErrs, errs...)
 	}
 
+	if errs := m.validateFleetsMember(old); len(errs) > 0 {
```

Comment:
Do we need to do any validation for AzureManagedControlPlaneTemplate?

Reply:
We don't need validation for the template since the name is the only thing we need to validate, and we don't specify the name in the template.